code/modules/paperwork/photography.dm

```diff
@@ -325,7 +325,7 @@ var/list/SpookyGhosts = list("ghost","shade","shade2","ghost-narsie","horror","s
 	to_chat(user, "<span class='notice'>[pictures_left] photos left.</span>")
 	icon_state = icon_off
 	on = 0
-	if(user.mind && !(user.mind.assigned_role == "Chaplain"))
+	if(user.mind && user.mind.assigned_role == "Chaplain")

```

Comment:
this also needs to check for `see_ghosts`, because regular cameras shoudln't be spooping people.